File: Safari-611.1.10/LayoutTests/http/tests/xmlhttprequest/resources/access-control-response-with-expose-headers.php
<?php
    header("Access-control-max-age: 0");
    header("Access-control-allow-origin: *");
    header("X-foo: BAR");
    header("X-TEST: TEST");
    header("Access-Control-Expose-Headers: x-Foo");
    header("Content-Type: text/html");

    print "echo"
?>

File: LayoutTests/http/tests/xmlhttprequest/resources/access-control-response-with-expose-headers.py
#!/usr/bin/env python3
import os
import sys
sys.stdout.reconfigure(newline="")  # prevent windows \n -> \n\r conversion

sys.stdout.write(
    'Content-Type: text/html\r\n'
    'Access-control-max-age: 0\r\n'
    'Access-control-allow-origin: *\r\n'
    'X-foo: BAR\r\n'
    'X-TEST: TEST\r\n'
    'Access-Control-Expose-Headers: x-Foo\r\n'
    'Content-Type: text/html\r\n'
    '\r\n'
    'echo\n'
)
